honssh/output_handler.py

```diff
@@ -233,10 +240,12 @@ def channel_opened(self, channel_id, channel_name):
         plugins.run_plugins_function(self.loaded_plugins, 'channel_opened', True, channel)
 
     def channel_closed(self, channel):
+        self._channel_closed(channel.uuid)
```

Comment:
_channel_closed takes an string and is called only from within the class while channel_closed takes an channel object and gets called from outside.
This is needed as we don't have the channel objects in connection_lost. So the only possibility is to get all channel uuids from connections and close them one by one.